content/filters/additional_filters.md

```diff
@@ -0,0 +1,155 @@
+# Additional Filters
+
+## url_encode
+
+Check does source can be encode for URI link.
+
+#### Input
+
+~~~ liquid
+  <!-- Rental Headline = "Super Headline" -->
+    {{ rental.headline | url_encode }}
+~~~
+
+#### Output
+
+~~~
+  Super%20Headline
+~~~
+
+## rental_photo_url
+
+Load photo with given size.
```

Comment:
Insterts `img` tag with link to the given size.
List of all of the sizes would be good